api/versioning/BUILD

```diff
@@ -16,16 +16,15 @@ proto_library(
         "//envoy/config/common/matcher/v3:pkg",
         "//envoy/config/core/v3:pkg",
         "//envoy/config/endpoint/v3:pkg",
-        "//envoy/config/filter/thrift/router/v2alpha1:pkg",
         "//envoy/config/grpc_credential/v3:pkg",
-        "//envoy/config/health_checker/redis/v2:pkg",
+        "//envoy/config/health_checker/redis/v3:pkg",
         "//envoy/config/listener/v3:pkg",
         "//envoy/config/metrics/v3:pkg",
         "//envoy/config/overload/v3:pkg",
         "//envoy/config/ratelimit/v3:pkg",
         "//envoy/config/rbac/v3:pkg",
-        "//envoy/config/resource_monitor/fixed_heap/v2alpha:pkg",
-        "//envoy/config/resource_monitor/injected_resource/v2alpha:pkg",
+        "//envoy/config/resource_monitor/fixed_heap/v3:pkg",
```

Comment:
@ankatare I think we need to have both v2 and v3 variants in the v3 protos, changing this should fix your CI error.

Reply:
Ok. let me do it now. thanks for review !!!

Reply:
@htuch sorry. i am bit confused here. as there are only 2 categories in BUILD file as active_protos and frozen_protos. 
what do you mean by "have both v2 and v3 variants in the v3 protos"

Reply:
Sorry, I meant to say "have both v2 and v3 variants in the active protos". Good point.

Reply:
i think, this already taken care.